src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -223,6 +220,9 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals
     task = client.get(resource_group_name, registry_name, task_name)
     step = task.step
     if isinstance(step, DockerBuildStep):
+        if file.endswith(ALLOWED_TASK_FILE_TYPES):
+            raise CLIError("File for docker build step cannot have folliwng suffix: {} "
```

Comment:
nit: extra trailing space; typo `following`.
Might right better as "File for Docker build step has an invalid suffix: {}. The following suffixes are not allowed: {}"

Same for the other `raise CLIError` below